tests/integration.rs

```diff
@@ -736,3 +736,53 @@ fn test_expire() -> Result<()> {
 
     Ok(())
 }
+
+#[test]
```

Comment:
Question: does this test (`test_alloc_data_type `) gets executed when you run the `cargo test` command?

Reply:
Yes